SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -8,19 +8,122 @@
 import SwiftUI
 
 struct ContentView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        
+        
+        static let alertButtonsAndLogicText = "Пример Alert c 2 кнопками и логикой"
+        static let LabelAlertText = "Пример Alert"
+        static let labelActionSheetText = "Пример ActionSheet"
+        static let labelActionSheetWithButtonText = "Пример ActionSheet c кнопками и логикой"
+        static let buttonsLabelText = "Показать"
+        static let alertOneTitleText = "Пример алерта"
+        static let alertTwoTitleText = "Изменить текст кнопки?"
+        static let alertTwoDefaultButtonTitle = "Меняем"
+        static let newValueText = "Новый текст"
+        static let actionSheetTitleText = "А вот он я"
+        static let actionSheetAlarmTitle = "Внимание!"
+        static let actionSheetTwoMessageText = "Сейчас будем менять цвета!"
+        static let actionSheetTwoDefaultButtonText =  "меняем цвет надписи"
+        static let cornerRadiusNumber: CGFloat = 5
+        static let paddingNumber: CGFloat = 15
+    }
+    
+    @State var isAlertExample = false
+    @State var isTwoButtonAlertExample = false
+    @State var isActionSheetExample = false
+    @State var isActionSheetTwoExample = false
+    @State var oldValueText = Constants.labelActionSheetWithButtonText
+    @State var textColor = Color.black
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            Text(Constants.LabelAlertText).foregroundColor(.black)
+            showAlert().padding(Constants.paddingNumber)
+               
+            Text(oldValueText)
+            showTwoButtonAlert().padding(Constants.paddingNumber)
+            
+            Text(Constants.labelActionSheetText)
+            showActionSheet().padding(Constants.paddingNumber)
+            
+            Text(Constants.labelActionSheetWithButtonText).foregroundColor(textColor)
+            showTwoAсtionSheet().padding(Constants.paddingNumber)
+        }
+    }
+    
+    fileprivate func showAlert() -> some View {
```

Comment:
```suggestion
    private var showAlertView: some View {
```
Если нет параметров то делаем вместо метода вычисляемые свойства